qfieldsync/gui/package_dialog.py

```diff
@@ -70,10 +70,15 @@ def __init__(self, iface, project, offline_editing, parent=None):
         self.iface = iface
         self.offliner = QgisCoreOffliner(offline_editing=offline_editing)
         self.project = project
+        self.original_project_path = self.project.fileName()
         self.qfield_preferences = Preferences()
         self.dirsToCopyWidget = DirsToCopyWidget()
         self.__project_configuration = ProjectConfiguration(self.project)
-        self.project_lbl.setText(get_project_title(self.project))
+        self.original_project_title = project.title()
+        self.packaged_project_title.setText(self.original_project_title)
+        self.packaged_project_name.setText(self.project.baseName())
```

Comment:
What if the project we are trying to package is not saved yet? Imagine you open a new blank project, set all the layers, config etc and then press "package project" without ever saving it to the disk?